commands/beautify.ml

```diff
@@ -0,0 +1,27 @@
+open Api
+open Kernel
+open Cmdliner
+
+let beautify config files =
+  let open Processor in
+  let after _ = Option.iter (fun (env, lc, e) -> Env.fail_env_error env lc e) in
```

Comment:
I suggest to not eta-reduce here because we don't understand over what you are doing the iteration and why it should fails.